flatgfa/src/cli/cmds.rs

```diff
@@ -334,7 +336,48 @@ pub fn bed_intersect(args: BEDIntersect) {
             let name: &BStr = bed2.get_name_of_entry(val);
             let start = val.start;
             let end = val.end;
-            println!("{}\t{}\t{}", name, start, end);
+            println!("{name}\t{start}\t{end}");
         }
     }
 }
+
+/// Print the contents of a compressed file of nucleotides
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "seq-import")]
+pub struct SeqImport {
+    /// the name of the file to import from
+    #[argh(positional)]
+    filename: String,
+}
+
+pub fn seq_import(args: SeqImport) {
+    let mmap = memfile::map_file(&args.filename);
+    let view = PackedSeqView::read_file(&mmap);
+    print!("{view}");
+}
+
+/// Compresses a sequence of nucleotides and exports it to a file
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "seq-export")]
+pub struct SeqExport {
+    /// the name of the file to export to
+    #[argh(positional)]
+    filename: String,
+}
+
+pub fn seq_export(args: SeqExport) {
+    let mut input = String::new();
+    std::io::stdin()
+        .read_to_string(&mut input)
+        .expect("Stdin read failure");
+
+    let vec: Vec<packedseq::Nucleotide> = input
+        .chars()
+        .filter(|c| !c.is_whitespace())
+        .map(packedseq::Nucleotide::from)
+        .collect();
```

Comment:
Not something to address in this PR, but it would be great to figure out how to avoid this `collect`. Relatedly, we should consider setting up some manner of benchmarking to measure the performance impact of that…